go.mod

```diff
@@ -168,6 +167,7 @@ require (
 	github.com/dustin/go-humanize v1.0.1 // indirect
 	github.com/emirpasic/gods v1.18.1 // indirect
 	github.com/erikgeiser/coninput v0.0.0-20211004153227-1c3628e74d0f // indirect
+	github.com/ettle/strcase v0.2.1-0.20230114185658-e5db6a6becf3
```

Comment:
Hi @jedevc, one thing I want to bring up is that this version is being pulled from a branch for `ettle/strcase` lib here.  https://github.com/ettle/strcase/pull/13

I know not ideal, but out of the other libraries that i tried, only this combination works as we expected. 

do you think we should fork this library until that PR is merged?

Reply:
on thinking more about it, I think we should use a fork of this library until the PR is merged.